### ISSUE TITLE
Simple error prevents import of ALAC files into beets

### ISSUE THREAD

**Issue by oscarcarlsson** [sampsyo/beets#1201]
I just noticed this when trying to import a few ALAC files into my new beets library:

	user configuration: /path/to/.config/beets/config.yaml
	data directory: /path/to/.config/beets
	Sending event: pluginload
	library database: /path/to/.config/beets/library.db
	library directory: /path/to/Music
	Sending event: library_opened
	Traceback (most recent call last):
	  File "/path/to/env/python-beets/bin/beet", line 9, in <module>
		load_entry_point('beets==1.3.9', 'console_scripts', 'beet')()
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/ui/__init__.py", line 935, in main
		_raw_main(args)
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/ui/__init__.py", line 925, in _raw_main
		subcommand.func(lib, suboptions, subargs)
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/ui/commands.py", line 872, in import_func
		import_files(lib, paths, query)
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/ui/commands.py", line 844, in import_files
		session.run()
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/importer.py", line 299, in run
		pl.run_parallel(QUEUE_SIZE)
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/util/pipeline.py", line 249, in run
		msg = self.coro.next()
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/importer.py", line 1070, in read_tasks
		task = task_factory.album([toppath], dir=toppath)
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/importer.py", line 997, in album
		items = map(self.read_item, paths)
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/importer.py", line 1015, in read_item
		return library.Item.from_path(path)
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/library.py", line 391, in from_path
		i.read(path)
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/library.py", line 443, in read
		mediafile = MediaFile(syspath(read_path))
	  File "/path/to/env/python-beets/local/lib/python2.7/site-packages/beets/mediafile.py", line 1344, in __init__
		if self.mgfile.codec and self.mgfile.codec.startswith('alac'):
	AttributeError: 'MP4' object has no attribute 'codec'

**The fix is trivial**, line 1344 in mediafile.py:

                if self.mgfile.info.codec and self.mgfile.info.codec.startswith('alac'):

Original state:

                if self.mgfile.codec and self.mgfile.codec.startswith('alac'):

This allows import of ALAC, iTunes bought AAC (256 kbit) and custom encoded AAC (varying kbit), but this is only after some happy testing with my 'fix'.

**Comment by claman** [comment]
I like the sound of this.

**Comment by sampsyo** [comment]
Thank you! Please see #1117.

**Comment by oscarcarlsson** [comment]
Not sure how I missed this other issue, I thought I had looked through the list. Sorry for this issue :)

**Comment by QuantumQweef** [comment]
welp. there's a few closed copies of this exact issue..and I hate to rain on anyones parade. But I still havn't been able to get ALAC files thru (several of the last few) source builds of beets.

Not sure if I should open the issue back up, so hopefully someone'll see this and weigh in on what they're experiencing.

**Comment by sampsyo** [comment]
Are you seeing the same error as in #1117 on the latest source version of beets? If so, please comment there. And if it's a different error, please do open a new issue.

**Comment by QuantumQweef** [comment]
**and of course I have been collecting all the error logs & and anything else that I'll upload if need be
